File: PUGD-api/graphql/types/AcquisitionTypes/DeliveryLine.go
package AcquisitionTypes

import "github.com/graphql-go/graphql"

var DeliveryLineType = graphql.NewObject(graphql.ObjectConfig{
	Name: "DeliveryLineType",
	Fields: graphql.Fields{
		"_id": &graphql.Field{
			Type: graphql.ID,
		},
		"orderline": &graphql.Field{
			Type: graphql.ID,
		},
		"order": &graphql.Field{
			Type: graphql.ID,
		},
		"isbn": &graphql.Field{
			Type: graphql.String,
		},
		"date": &graphql.Field{
			Type: graphql.String,
		},
		"title": &graphql.Field{
			Type: graphql.String,
		},
		"newquantity": &graphql.Field{
			Type: graphql.Int,
		},
		"remainingquantity": &graphql.Field{
			Type: graphql.Int,
		},
	},
})

File: PUGD-frontend/graphql/mutations/acquisition/deliveryline.js
import gql from "graphql-tag";
const InsertDeliveryLine = gql`
  mutation(
    $_id: String!
    $orderline: String!
    $isbn: String!
    $title: String
    $date: String
    $newquantity: Int
    $remainingquantity: Int
    $order: String!
  ) {
    insertDeliveryLine(
      _id: $_id
      order: $order
      orderline: $orderline
      isbn: $isbn
      title: $title
      date: $date
      newquantity: $newquantity
      remainingquantity: $remainingquantity
    )
  }
`;
module.exports = {
  InsertDeliveryLine,
};
